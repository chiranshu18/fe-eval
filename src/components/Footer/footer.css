.footer{
    display: flex;
    flex-direction: row;
    align-items: center;
    position: fixed;
    bottom: 0;
    background-color: black;
    color: white;
    gap: 10px;
    padding-top: 30px;
    padding-left: 10vw;
    padding-right: 10vw;
    width: 100%;
}

.pink{
    background-color: rgb(171, 69, 86) ;
    border-radius: 10px;
    width: 50px;
    height: 50px;
}

.blue{
    background-color:  navy;
    border-radius: 10px;
    width: 50px;
    height: 50px;
}

.gray{
    background-color: #9B9999 ;
    border-radius: 10px;
    width: 50px;
    height: 50px;
}