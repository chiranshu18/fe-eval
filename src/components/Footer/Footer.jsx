import React from 'react'
import './footer.css'

const Footer = () => {
  return (
    <div className='footer'>
      <h2>THEMES</h2>
      <div className='pink'></div>
      <div className='blue'></div>
      <div className='gray'></div>
    </div>
  )
}

export default Footer