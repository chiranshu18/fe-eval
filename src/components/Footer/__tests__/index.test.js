import Footer from '../Footer'
import { render , screen} from "@testing-library/react";

describe('footer', () => {
    
    it('should display footer text when rendered', () => {
        render(<Footer />);
        expect(screen.getByText('THEMES')).toBeTruthy();
    });

    it('Should render correctly', () => {
        const {asFragment} = render(<Footer />)
        expect(asFragment()).toMatchSnapshot();
    })
});