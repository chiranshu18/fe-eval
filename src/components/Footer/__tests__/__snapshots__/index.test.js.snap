// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <h2>
      THEMES
    </h2>
    <div
      class="pink"
    />
    <div
      class="blue"
    />
    <div
      class="gray"
    />
  </div>
</DocumentFragment>
`;

exports[`navbar Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <h2>
      THEMES
    </h2>
    <div
      class="pink"
    />
    <div
      class="blue"
    />
    <div
      class="gray"
    />
  </div>
</DocumentFragment>
`;
